styx-scheduler-service/src/test/java/com/spotify/styx/docker/LabelValueTest.java

```diff
@@ -0,0 +1,104 @@
+/*-
+ * -\-\-
+ * Spotify Styx Scheduler Service
+ * --
+ * Copyright (C) 2016 - 2020 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+package com.spotify.styx.docker;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import com.spotify.styx.util.ClassEnforcer;
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+@RunWith(JUnitParamsRunner.class)
+public class LabelValueTest {
+
+  private static final int KUBERNETES_LABEL_MAX_LENGTH = 63;
+  private static final int DIGEST_SUFFIX_LENGTH = 7;
+  private static final int PREFIX_MAX_LENGTH = KUBERNETES_LABEL_MAX_LENGTH - DIGEST_SUFFIX_LENGTH;
+
+  @Test
+  @Parameters({
+      "",
+      "a",
+      "Test-value_including.0-and-9",
+  })
+  public void shouldHandleSimpleCases(String input) {
+    var output = LabelValue.normalize(input);
+    assertThat(output, is(input));
+  }
+
+  @Test
+  public void shouldNotModifySimple63ByteValue() {
+    var input = repeat("0", KUBERNETES_LABEL_MAX_LENGTH);
+    var output = LabelValue.normalize(input);
+    assertThat(output, is(input));
+  }
+
+  @Test
+  public void shouldSuffixChangedShortValue() {
+    var input = "spotify#com";
+    var output = LabelValue.normalize(input);
+    var expected = "spotifycom3de352c";
+    assertThat(output, is(expected));
+  }
+
+  @Test
+  public void shouldTruncateAndSuffixLongValue() {
+    var input = repeat("0", KUBERNETES_LABEL_MAX_LENGTH + 1);
+    var output = LabelValue.normalize(input);
+    var expected = repeat("0", PREFIX_MAX_LENGTH) + "60e05bd";
+    assertThat(output, is(expected));
+  }
+
+  @Test
+  public void shouldProperlyHandleMultiByteValue() {
```

Comment:
I had to go back and read the code in normalize to understand what Properly really mean.
What about renaming the test to: `shouldRemoveMultiByteValueAndSufix`?

Reply:
Didn't want to touch borrowed code too much, but I'm fine changing it.

Reply:
I renamed it.